fix: Rollback bump to llama-stack-client

derekhiggins opened the pull request:
The bump to startlet also bumped lls client
this wasn't needed and we suspect is causing build problems downstream.

leseb commented on README.md:
```diff
@@ -1,177 +1,11 @@
+ODH 2.22 is the final version that uses this repository. We do not expect to create new branches in this repository.
 
-<details>
+Future versions of RH OpenShift AI will use another system for `llama-stack` maintenance.
```
This is not true anymore, please remove :)

leseb reviewed (commented):
Ok, this is almost ready, thanks - we need to do something similar to the release branches where we only keep the redhat-distribution directory. Would you mind taking care of this?

ktdreyer commented:
Artemy is handling this now in https://github.com/opendatahub-io/llama-stack/pull/27

derekhiggins committed:
fix: Rollback bump to llama-stack-client

The bump to startlet also bumped lls client
this wasn't needed and we suspect is causing build problems
downstream.

Signed-off-by: Derek Higgins <derekh@redhat.com>

leseb commented on pyproject.toml:
```diff
@@ -7,7 +7,7 @@ required-version = ">=0.7.0"
 [project]
 name = "llama_stack"
-version = "0.2.22.1+rhai0"
+version = "0.2.22.2+rhai0"
```
is this intended?

derekhiggins replied:
Yes, if we merge this then I'll create a v0.2.22.2+rhai0 tag  so we can build form that

leseb replied:
🤦🏻 it's getting late, was thinking of another repo. this is obviusly good

leseb commented on pyproject.toml:
```diff
@@ -31,14 +31,14 @@ dependencies = [
     "pydantic>=2.11.9",
     "rich",
-    "starlette>=0.49.1",
+    "starlette==0.49.1",
```
why do we pin?

derekhiggins replied:
I was trying to minimise the number of changes in uv.lock (less chance of more unintended consequences )
